packages/m/mhook/xmake.lua

```diff
@@ -0,0 +1,20 @@
+package("mhook")
+
+    set_homepage("https://github.com/martona/mhook")
+    set_description("A Windows API hooking library ")
+
+    set_urls("https://github.com/apriorit/mhook/archive/$(version).zip")
+
+    add_versions("2.5.1", "37e7d65422a770b0a28835b4fe557823278f7118ff4a399d7acf345c8db318e5")
+
+    add_patches("2.5.1", "https://github.com/apriorit/mhook/commit/5ccb00a9c89280bfff7ce595873a9923415172a7.patch",
+                        "56561718ccf05c8c42fff05e6531cfa525cf93e0c9fa3bd226e74ef19eae1a1f")
+
+    on_install("windows", function (package)
+        local configs = {}
+        import("package.tools.cmake").install(package, configs, {buildir = "build"})
```

Comment:
If you do not use the build directory `, {buildir = "build"}`, you can remove it.